extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -374,6 +390,14 @@ export class SubscriptionService {
     return readerId;
   }
 
+  /**
+   * @param {string} serviceId
+   * @return {!Promise<?string>}
+   */
+  getEncryptedDocumentKey(serviceId) {
+    this.cryptoHandler_.getEncryptedDocumentKey(serviceId);
```

Comment:
Missing `return`? Also, it looks like `cryptoHandler.getEncryptedDocumentKey()` returns a non-promise result?